python/tempo/tsdf.py

```diff
@@ -517,6 +519,32 @@ def resample(self, freq, func=None, metricCols = None, prefix=None, fill = None)
     enriched_tsdf = rs.aggregate(self, freq, func, metricCols, prefix, fill)
     return(enriched_tsdf)
 
+  def interpolate(self, target_cols: List[str], sample_freq: str, sample_func: str, fill: str, ts_col: str = None, partition_cols: List[str]=None):
```

Comment:
Suggestions: 

Make an option for adding the 'is_interpolated' columns. I see this being really valuable but if there are 90 columns, this could get really cumbersome to maintain in the table. 
Make an option for forcing a resample - in this case, the user can supply an argument for freq/func. Otherwise, we assume it's done and interpolate won't do much.

Reply:
Changed this to a boolean that can be set when calling `interpolate` by default it's set to not show.